js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -1329,6 +1329,119 @@ function processQuery (query, explain) {
+          let OLD="";
+          if (node.hasOwnProperty('inVariable')) {
+            collectionVariables[node.inVariable.id] = node.collection;
+            OLD = `${keyword('WITH')} ${variableName(node.inVariable)} `;
+          }
+          let indexRef;
+          let keyCondition = "";
+          let filterCondition;
+          if (node.hasOwnProperty('key')) {
+            keyCondition = `{ _key: ${variable(JSON.stringify(node.key))} } `;
+            indexRef = `${variable(JSON.stringify(node.key))}`;
+            filterCondition = `${variable('doc._key')} == ${variable(JSON.stringify(node.key))}`;
+          } else if (node.hasOwnProperty('inVariable')) {
+            keyCondition = `${variableName(node.inVariable)} `;
+            indexRef = `${variableName(node.inVariable)}`;
+          } else {
+            keyCondition = "WTFFF?";
+            indexRef = "WTF?";
+          }
+          if (node.hasOwnProperty('indexes')) {
+            node.indexes.forEach(function(idx, i) { iterateIndexes(idx, i, node, types, indexRef); });
+          }
+          let FORSTATEMENT="";
+          if (node.replaceIndexNode) {
+            FORSTATEMENT=`${keyword('FOR')} ${variable('doc')} ${keyword('IN')} ${collection(node.collection)} ${keyword('FILTER')} ${filterCondition} `;
+            keyCondition = `${variable('doc')} `;
+          }
+          return `${FORSTATEMENT}${keyword('REPLACE')} ${keyCondition}${OLD}${keyword('IN')} ${collection(node.collection)}`;
+        }
+        case 'RemoveNode': {
+          modificationFlags = node.modificationFlags;
+          if (node.hasOwnProperty('inVariable')) {
+            collectionVariables[node.inVariable.id] = node.collection;
+          }
+          let indexRef;
+          let keyCondition;
+          let filterCondition;
+          if (node.hasOwnProperty('key')) {
+            keyCondition = `{ _key: ${variable(JSON.stringify(node.key))} } `;
+            indexRef = `${variable(JSON.stringify(node.key))}`;
+            filterCondition = `${variable('doc._key')} == ${variable(JSON.stringify(node.key))}`;
+          } else if (node.hasOwnProperty('inVariable')) {
+            keyCondition = `${variableName(node.inVariable)} `;
+            indexRef = `${variableName(node.inVariable)}`;
+          } else {
+            keyCondition = "WTFFF?";
+            indexRef = "WTF?";
+          }
+          if (node.hasOwnProperty('indexes')) {
+            node.indexes.forEach(function(idx, i) { iterateIndexes(idx, i, node, types, indexRef); });
+          }
+          let FORSTATEMENT="";
+          if (node.replaceIndexNode) {
+            FORSTATEMENT=`${keyword('FOR')} ${variable('doc')} ${keyword('IN')} ${collection(node.collection)} ${keyword('FILTER')} ${filterCondition} `;
+            keyCondition = `${variable('doc')} `;
+          }
+          return `${FORSTATEMENT}${keyword('REMOVE')} ${keyCondition}${keyword('IN')} ${collection(node.collection)}`;
+        }
+        }
```

Comment:
indentation is either not right here, or one of the closing `}` brackets is too much (at least when viewing this diff in no-whitespace mode)